src/File/FileHelper.php

```diff
@@ -52,7 +53,14 @@ public function absolutizePath(string $path): string
 	/** @api */
 	public function normalizePath(string $originalPath, string $directorySeparator = DIRECTORY_SEPARATOR): string
 	{
-		$isLocalPath = $originalPath !== '' && $originalPath[0] === '/';
+		$isLocalPath = false;
+		if ($originalPath !== '') {
+			if ($originalPath[0] === '/') {
+				$isLocalPath = true;
+			} elseif (strlen($originalPath) >= 3 && $originalPath[0] === 'C' && $originalPath[1] === ':' && $originalPath[2] === '\\') {
+				$isLocalPath = true;
```

Comment:
on windows we now take this fast path and prevent PCRE usage

Reply:
What about other drive letters?

Reply:
Yeah, but I feel like there has to be a better way other than comparing letters as if they were numbers :)

Reply:
wdyt about using `ord()`? :)

to be honest, I don't like both ways. using regex would work, but we wanne be fast, right? :)

Reply:
ok, another idea.  now using `range()`